README.rst

```diff
@@ -36,18 +36,22 @@ Usage
 =====
 
 Magic-Folder is configured via the ``magic-folder`` command-line tool.
-This tool manipulates the ``magic_folders.yaml`` file in a Tahoe-LAFS' node's private area.
-See the configuration documentation for full details.
 
-Once Magic-Folder is configured,
-functionality is provided by running a long-lived magic-folder side-car for the Tahoe-LAFS node.
-Start the side-car process using the ``magic-folder`` command line too::
+Magic-Folder configuration is kept in a directory. Inside this directory is a database for global configuration and sub-directories to track state and temporary space for each actual magic-folder including a configuration database. All databases are SQLite.
```

Comment:
Let's use semantic linefeeds for magic-folder docs.

Reply:
I kind of thought I was .. I guess I got the granularity wrong? Are you asking for a linefeed per "."?

Reply:
Yea, I think that's the minimum number of linefeeds I interpret "semantic linefeeds" as calling for.  Sometimes I also put them in after `,` or other intra-sentence punctuation.  It starts to get a bit more subjective when you go beyond `.` (so I wouldn't generally ask for *more* newlines than that in a review).